config/webconfig.go

```diff
@@ -52,6 +52,7 @@ func WebConfig(r *http.Request) (*HeplifyServer, error) {
 		webSetting.DBConfTable = "homer_config"
 	}
 	webSetting.DBAddr = r.FormValue("DBAddr")
+	webSetting.DBAddr = r.FormValue("DBSSLMode")
```

Comment:
Are you *sure* the `DBSSLMode` FormValue should be mapped to the identically named setting `webSetting.DBAddr`??

Reply:
Good catch! Would you like to send a PR?

Reply:
@ZionDials u fix dis?